boards/microbit_v2/README.md

```diff
@@ -112,21 +100,36 @@ $ make flash
 $ make flash-debug
 ```
 
-### Flashing app
+## Managing applications
+
+Make sure you have flashed the
+[Tock Bootloader](https://github.com/tock/tock-bootloader) to the board.
+
+Make sure you have [Tockloader](https://github.com/tock/tockloader) installed
+(version 1.8.0 is required to automatically detect and handle Microbit
+applications correctly).
+
+Please refer to the [tockloader](https://github.com/tock/tockloader)
+documentation to install apps.
 
-Please refer to the [tockloader](https://github.com/tock/tockloader) documentation to flash apps.
 
-With bootloader
+To install apps by automatically detecting the Microbit v2:
 ```bash
 $ tockloader install app.tab
 ```
 
-Without bootloader
+To manually inform Tockloader of the board configuration:
 ```bash
-$ tockloader --openocd --board microbit_v2 --bundle-apps install app.tab
+$ tockloader install --openocd --board microbit_v2 --bundle-apps app.tab
 ```
 
-> `--bundle-apps` seems to be needed due to an [openocd issue](https://github.com/tock/tockloader/issues/67)
+> `--bundle-apps` seems to be needed due to an [openocd issue](https://github.com/tock/tockloader/issues/67).
+
+> If you are using an older version of Tockloader, add `--page-size 512` at the end of the command.
+>
+> ```bash
+> $ tockloader ... --page-size 512
+> ```
```

Comment:
@bradjc Is there a way to manually specify minimum app size for Tockloader? Or should I leave `--bundle-apps` here? It matters for people not on the newest Tockloader update.

Reply:
No.